internal/plugin/connectors/http/generic/connector.go

```diff
@@ -14,6 +15,40 @@ type Connector struct {
 	config *config
 }
 
+func addOAuth1Header(c *Connector, credentialsByID connector.CredentialValuesByID, r *gohttp.Request) error {
+	oAuth1Params, _ := renderTemplates(c.config.OAuth1Secrets, credentialsByID)
+	if len(oAuth1Params) > 0 {
+		oauthHeader, err := oauth1.CreateOAuth1Header(oAuth1Params, r)
+		if err != nil {
+			return fmt.Errorf("failed to create oAuth1 'Authorization' header: %s", err)
+		} else if len(r.Header.Get("Authorization")) > 0 {
```

Comment:
We changed how this is handled to kill Secretless if someone declares an `Authorization` header and `oauth1` in the config since the header will be overwritten anyways. Must've forgot to change the README, fixing now